### PR TITLE
Simpler (and correct) example for showing how to declare foreign keys.

### PR DESCRIPTION
**Pull request by nicholasf**
Impact: Trivial.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1091,7 +1091,7 @@ type Product struct {
 // 2nd param : destination table(id)
 // 3rd param : ONDELETE
 // 4th param : ONUPDATE
-db.Model(&User{}).AddForeignKey("user_id", "destination_table(id)", "CASCADE", "RESTRICT")
+db.Model(&User{}).AddForeignKey("role_id", "roles", "CASCADE", "RESTRICT")
```
Comment (jinzhu) [inline README.md]: I think `roles(id)` is required?

Comment (nicholasf) [inline README.md]: No, this is the real world example it's based on (which works).

```
	db.Model(property).AddForeignKey("id", "products", "CASCADE", "RESTRICT")
```

Comment (nicholasf) [inline README.md]: Hi,

I don't believe it is. Here's a real world example:

```
	db.Model(property).AddForeignKey("product_id", "products", "CASCADE", "RESTRICT")

```

```
phoenix_test=# \d properties;
                                    Table "public.properties"
   Column   |           Type           |                        Modifiers                        
------------+--------------------------+---------------------------------------------------------
 id         | integer                  | not null default nextval('properties_id_seq'::regclass)
 created_at | timestamp with time zone | 
 updated_at | timestamp with time zone | 
 deleted_at | timestamp with time zone | 
 product_id | integer                  | 
 value_id   | integer                  | 
 value_type | character varying        | 
Indexes:
    "properties_pkey" PRIMARY KEY, btree (id)
Foreign-key constraints:
    "properties_product_id_foreign" FOREIGN KEY (product_id) REFERENCES products(id) ON UPDATE RESTRICT ON DELETE CASCADE

phoenix_test=# 

```

**Comment by jinzhu** [comment]
Then it looks good to me, thank you for your pull request.

(sorry for the delay, i am still on vocation)

**Comment by nicholasf** [comment]
No probs. :)

**Pull request merged**